api-reference/10 UI Widgets/dxDataGrid/1 Configuration/editing/mode.md

```diff
@@ -7,7 +7,7 @@ The following list points out the differences in editing modes.
 - [**Row**](/concepts/05%20Widgets/DataGrid/20%20Editing/10%20User%20Interaction/10%20Row%20Mode.md '/Documentation/Guide/Widgets/{WidgetName}/Editing/#User_Interaction/Row_Mode')       
 A user edits one row at a time. The widget saves changes when the row leaves the editing state. See [demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RowEditingAndEditingEvents).
 - [**Batch**](/concepts/05%20Widgets/DataGrid/20%20Editing/10%20User%20Interaction/30%20Batch%20Mode.md '/Documentation/Guide/Widgets/{WidgetName}/Editing/#User_Interaction/Batch_Mode')     
-A user edits data cell by cell. The widget does not save changes until a user clicks the global *"Save"* button. See [demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/{WidgetName}/BatchEditing).
+A user edits data cell by cell. The widget does not save changes until a user clicks the global *"Save"* button. See these two demos that edit data stored locally and remotely: [Batch Editing](https://js.devexpress.com/Demos/WidgetsGallery/Demo/{WidgetName}/BatchEditing) and [Batch Update Request](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/BatchUpdateRequest/jQuery/Light/).
```

Comment:
Let's do this as follows:

> View the following demos: [Batch Editing] | [Batch Update Request]

And let's make similar changes to the rest of the items on this list.

One more thing: demo links shouldn't contain this part: `jQuery/Light/`